### ISSUE TITLE
Dockerfile is buggy

### ISSUE THREAD

**Issue by unton83** [kivy/buildozer#1798]
### Description

Please, refer to https://buildozer.readthedocs.io/en/latest/installation.html for installation tips for Dockerfile.
1. We need to use stable LTS, not **latest** one.
`FROM ubuntu:22.04`
2. Additional packages to install: `libltdl-dev`, `wget`, `vim`.
3. We don't need to do `COPY --chown=user:user . ${SRC_DIR}`, because we run container with `--volume "$PWD":/home/user/hostcwd`.
4. We need to install dependencies and buildozer (**it is missing** in current Dockerfile) running:
```
RUN pip3 install --user --upgrade Cython==0.29.33 virtualenv   # without dir
RUN pip3 install --user --upgrade buildozer
RUN echo "export PATH=$PATH:~/.local/bin/" >> ~/.bashrc
```
5. I think, entrypoint can be `["/bin/bash"]` not `["buildozer"]`, because, when we run container with `--volume "$HOME/.buildozer":/home/user/.buildozer` and `-it`, we can edit some code with `vim`, or do `buildozer appclean` or something else.

**Comment by manatlan** [comment]
Yes I confirm ;-)

the DockerFile doesn't work AS IS on an ubuntu 24.04 (lts) ...

need to use "FROM ubuntu:22.04" !

**Comment by misl6** [comment]
#1804 merged, anything else we should fix?

**Comment by unton83** [comment]
I'm new in github issues tracker. Do i need to do something with **awaiting-reply** label? Do i need to close the issue?

**Comment by misl6** [comment]
If the issue got fixed via #1804, please close the issue.
If you think we should fix something else, feel free to propose changes via a PR or here.

`awaiting-reply` is an automated label used to close issues after a period of inactivity.

**Comment by unton83** [comment]
No, my issue is wider, not just ubuntu version. Nothing else, but only things, that i mentioned.
